packaging/requirements/requirements-azure.txt

```diff
@@ -13,4 +13,5 @@ azure-mgmt-keyvault>=0.40.0,<0.41
 azure-mgmt-batch>=4.1.0,<5
 azure-mgmt-sql>=0.7.1,<0.8
 azure-mgmt-web>=0.32.0,<0.33
-azure-mgmt-containerservice>=1.0.0
\ No newline at end of file
+azure-mgmt-containerservice>=1.0.0
```

Comment:
Beware that latest is 2.0.0. You should change this to >=2.0.0,3.0.0 and check you're ok with 2.0.0

Reply:
this PR is not touching containerservice at all, so it's better not to change it here